tests/analyser/analyser.cpp

```diff
@@ -1060,3 +1060,16 @@ TEST(Analyser, unsuitablyConstrainedNlaSystem)
 
     EXPECT_EQ(libcellml::AnalyserModel::Type::UNSUITABLY_CONSTRAINED, analyser->analyserModel()->type());
 }
+
+TEST(Analyser, rearrangeAlgebraicEquation)
+{
+    auto parser = libcellml::Parser::create();
+    auto model = parser->parseModel(fileContents("analyser/unarranged_algebraic_equation.cellml"));
+
+    EXPECT_EQ(size_t(0), parser->issueCount());
+
+    auto analyser = libcellml::Analyser::create();
+    analyser->analyseModel(model);
```

Comment:
It's mainly me, but I like to have an empty line between a declaration and some consuming code.